frontend/src/scenes/web-analytics/WebAnalyticsFilters.tsx

```diff
@@ -1,36 +1,80 @@
 import { DateFilter } from 'lib/components/DateFilter/DateFilter'
 import { FilterBar } from 'lib/components/FilterBar'
+import { PropertyFilters } from 'lib/components/PropertyFilters/PropertyFilters'
+import { isEventPersonOrSessionPropertyFilter } from 'lib/components/PropertyFilters/utils'
 import { FEATURE_FLAGS } from 'lib/constants'
 import { LemonSegmentedSelect } from 'lib/lemon-ui/LemonSegmentedSelect'
-import { IconMonitor } from 'lib/lemon-ui/icons/icons'
+import { IconLink, IconMonitor, IconWithCount } from 'lib/lemon-ui/icons/icons'
 import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
+import { copyToClipboard } from 'lib/utils/copyToClipboard'
 import MaxTool from 'scenes/max/MaxTool'
 import { urls } from 'scenes/urls'
+import { userLogic } from 'scenes/userLogic'
 
 import { ReloadAll } from '~/queries/nodes/DataNode/Reload'
-import { PropertyMathType } from '~/types'
+import { AvailableFeature, PropertyMathType } from '~/types'
 
 import { PathCleaningToggle } from './PathCleaningToggle'
 import { TableSortingIndicator } from './TableSortingIndicator'
-import { WebAnalyticsLiveUserCount } from './WebAnalyticsLiveUserCount'
 import { WebConversionGoal } from './WebConversionGoal'
-import { WebPropertyFilters } from './WebPropertyFilters'
+import {
+    WEB_ANALYTICS_PROPERTY_ALLOW_LIST,
+    WebPropertyFilters,
+    getWebAnalyticsTaxonomicGroupTypes,
+} from './WebPropertyFilters'
 import { ProductTab } from './common'
 import { webAnalyticsLogic } from './webAnalyticsLogic'
 
+const CondensedFilterBar = ({ tabs }: { tabs: JSX.Element }): JSX.Element => {
+    const {
+        dateFilter: { dateTo, dateFrom },
+    } = useValues(webAnalyticsLogic)
+    const { setDates } = useActions(webAnalyticsLogic)
+
+    return (
+        <FilterBar
+            top={tabs}
+            left={
+                <>
+                    <ReloadAll iconOnly />
+                    <DateFilter allowTimePrecision dateFrom={dateFrom} dateTo={dateTo} onChange={setDates} />
+                    <WebAnalyticsCompareFilter />
+                </>
+            }
+            right={
+                <>
+                    <ShareButton />
+                    <WebVitalsPercentileToggle />
+                    <FiltersPopover />
+                    <WebAnalyticsDeviceToggle />
+                    <WebAnalyticsDomainSelector />
+                    <TableSortingIndicator />
+                </>
+            }
+        />
+    )
+}
+
 export const WebAnalyticsFilters = ({ tabs }: { tabs: JSX.Element }): JSX.Element => {
     const {
         dateFilter: { dateTo, dateFrom },
         isPathCleaningEnabled,
     } = useValues(webAnalyticsLogic)
     const { setDates, setIsPathCleaningEnabled } = useActions(webAnalyticsLogic)
 
+    const { featureFlags } = useValues(featureFlagLogic)
+
+    if (featureFlags[FEATURE_FLAGS.CONDENSED_FILTER_BAR]) {
+        return <CondensedFilterBar tabs={tabs} />
```

Comment:
What about `CondensedWebAnalyticsFilterBar`? 

I am usually against repeating those, but our filters are different from most components here have the distinction already